### PR TITLE
Update ActionsColumns UI component documentation

### PR DESCRIPTION
**Pull request by serhiyzhovnir**
## Purpose of this pull request

This pull request (PR) updates the ActionsColumns UI component documentation.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-actionscolumn.html
- https://devdocs.magento.com/guides/v2.2/ui_comp_guide/components/ui-actionscolumn.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/grid/columns/actions.js
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/grid/columns/actions.js
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/templates/grid/cells/actions.html
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/templates/grid/cells/actions.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/ui_comp_guide/components/ui-actionscolumn.md [hunk 1]
```diff
@@ -7,88 +7,96 @@ The ActionsColumns component implements a table's column responsible for display
...
+            <settings>
+                <label translate="true">Actions</label>
+            </settings>
+        </actionsColumn>
+    </columns>
+    ...
+</listing>
+```
+
+The `Vendor\Module\Ui\Component\Listing\Column\Actions` class need to extend the [`Magento\Ui\Component\Listing\Columns\Column`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/Component/Listing/Columns/Column.php) class and add actions data via `prepareDataSource()` method.
```
Comment (rogyar) [inline guides/v2.2/ui_comp_guide/components/ui-actionscolumn.md]:
```suggestion
The `Vendor\Module\Ui\Component\Listing\Column\Actions` class needs to extend the [`Magento\Ui\Component\Listing\Columns\Column`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/Component/Listing/Columns/Column.php) class and add actions data via `prepareDataSource()` method.
```

**Comment by serhiyzhovnir** [comment]
Hi @rogyar 
The PR was adjusted.
Please, review it again.
Thanks

**Comment by dobooth** [comment]
running tests

**Comment by jeff-matthews** [comment]
@dobooth, don't forget to add a whatsnew statement for major updates.

**Pull request merged**